Ludicrous-Ducks/AngularTake2
# navbar routing working with login

# Description
navbar is working with routing 
# Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
Product search ticket #24 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation

[ ] There is new documentation in this pull request that must be reviewed..

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

cd <folder>
git checkout master  
git checkout master pull
git branch -r   
git checkout sm-navi
```

## Steps to Test
Outline the steps to test the code in this pull request.

1. Run Django server for Desktop app 
2. Run API for Django REST 
3. Run  HS (http server) for Angular mobile client
4. Check for navigation bar  and click links once you are logged in. 
5. The search and submit does not work yet .
